helm/templates/deployment.yaml

```diff
@@ -2,35 +2,58 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   labels:
-    app.component: {{ .Values.containers.service.name }}
-  name: {{ .Values.containers.service.name }}
+    app.component: {{ .Values.name }}
```

Comment:
do we need this `app.component` label? I've noticed it in some of our charts, and think it may have been copied from one of the starter charts. 

When you create a new helm chart, the only metadata it has is `name`.

Reply:
I think it's likely that it's just left over from some start code we originally used.  However removal causes deployment to fail.  I'm not sure if this is because it's using it to match existing deployments and would be fine if it never had one, or if it is required.  I'll leave in for now but investigate further.